controllers/cluster_controller.go

```diff
@@ -89,6 +97,48 @@ func (r *ClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 func (r *ClusterReconciler) reconcile(ctx context.Context, cluster *capiv1alpha3.Cluster) (ctrl.Result, error) {
 	r.Log.Debugf(ctx, "calling reconcile")
+
+	migrator, err := r.MigratorFactory.NewMigrator(*cluster)
```

Comment:
Feels like `NewMigrator` should take a pointer.

Reply:
+1

Reply:
done